tests/pkg_micro-ecc/tests/01-run.py

```diff
@@ -4,16 +4,18 @@
 from testrunner import run
 
 
-# Use a custom global timeout for slow hardware. On microbit (nrf51), the
-# test completes in 80s.
-TIMEOUT = 100
+# Use a custom global timeout for slow hardware. On ATmegas clocked at 8MHz
+# test completes in ~10 min
+TIMEOUT = 600
```

Comment:
I think there's no need to change this value since there's now a check on `Round x`. The global timeout variable is applied to all `child.expect` calls.
https://github.com/RIOT-OS/RIOT/blob/eafa381a06fed27bedc938c870567c68ceda84d3/dist/pythonlibs/testrunner/__init__.py#L25-L27

On atmega256rfr2-xpro, clocked at 16MHz, there's about 18s delay between each `Round x` display, so I guess 100s will be ok for almost all platforms.

Reply:
Done. Thanks for pointing out that the timeout is per `child.expect_*()`, not globally.